scripts/components/ZoomButtons/ZoomButtons.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './ZoomButtons.scss';
+
+export default class ZoomButtons extends React.Component {
+  /**
+   * @class
+   * @param {object} props React props.
+   */
+  constructor(props) {
+    super(props);
+    this.props = props;
+  }
+
+  /**
+   * React render function.
+   * @returns {object} JSX element.
```

Comment:
Using `object` here will make the code _less_ typesafe

```suggestion
   * @returns {JSX.Element} JSX element.
```

Reply:
`returns {object} JSX element.` is used on all (or mostly all) components in this repo, using `JSX.Element` instead og `object` creates warning "The type 'JSX' is undefined"

Reply:
What about `import('react').ReactElement`